Dockerfile

```diff
@@ -1,31 +1,10 @@
 FROM jwilder/nginx-proxy
-MAINTAINER [ "David Parrish <daveparrish@tutanota.com", "Yves Blusseau <90z7oey02@sneakemail.com>", "Hadrien Mary <hadrien.mary@gmail.com>" ]
 
-RUN apt-get update
-
-# Install python packages needed by simp_le
-RUN apt-get install -y -q --no-install-recommends python python-requests
-
-# Install python packages needed to build  simp_le
-RUN apt-get install -y -q --no-install-recommends git gcc libssl-dev libffi-dev python-dev python-pip
-
-# Get Let's Encrypt simp_le client source
-RUN git -C /opt clone https://github.com/kuba/simp_le.git
-
-WORKDIR /opt/simp_le
-# Upgrade setuptools
-RUN pip install -U setuptools
-# Install simp_le in /usr/local/bin
-RUN python ./setup.py install
-
-# Make house cleaning
-RUN rm -rf /opt/simp_le
-
-RUN apt-get autoremove -y git gcc libssl-dev libffi-dev python-dev python-pip
-
-RUN apt-get clean all
-RUN rm -r /var/lib/apt/lists/*
+MAINTAINER David Parrish <daveparrish@tutanota.com>
+MAINTAINER Yves Blusseau <90z7oey02@sneakemail.com>
+MAINTAINER Hadrien Mary <hadrien.mary@gmail.com>
 
 COPY . /app/
 
-WORKDIR /app/
+# Install simp_le program
+RUN chmod +rx /app/install_simp_le.sh && /app/install_simp_le.sh && rm -f /app/install_simp_le.sh
```

Comment:
I'm getting `/bin/sh: 1: /app/install_simp_le.sh: Text file busy` error testing on Vultr's servers. Here is the issue on Docker issues: https://github.com/docker/docker/issues/9547

The fix for me is to add sync like this:
``` diff
diff --git a/Dockerfile b/Dockerfile
index 27ced08..22dd65e 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -7,4 +7,4 @@ MAINTAINER Hadrien Mary <hadrien.mary@gmail.com>
 COPY . /app/
 
 # Install simp_le program
-RUN chmod +rx /app/install_simp_le.sh && /app/install_simp_le.sh && rm -f /app/install_simp_le.sh
+RUN chmod +rx /app/install_simp_le.sh && sync && /app/install_simp_le.sh && rm -f /app/install_simp_le.sh
```